src/pretalx/submission/permissions.py

```diff
@@ -62,7 +71,8 @@ def is_review_author(user, obj):
 rules.add_perm('submission.unconfirm_submission', can_be_unconfirmed & (is_speaker | is_orga))
 rules.add_perm('submission.remove_submission', can_be_removed & is_orga)
 rules.add_perm('submission.edit_submission', (can_be_edited & is_speaker) | is_orga)
-rules.add_perm('submission.review_submission', is_reviewer & ~is_speaker)
 rules.add_perm('submission.view_submission', is_speaker | is_orga | is_reviewer)
+rules.add_perm('submission.review_submission', is_reviewer & ~is_speaker)
```

Comment:
Reviews can no longer be edited after the talk has been accepted/rejected, that's nice. But new reviews can still be submitted, is that okay?

Reply:
I pushed a fix for this, thank you!